### DISCUSSION TITLE
最新版本，上传压缩包无法解压

### DISCUSSION THREAD

**Discussion by supspp** [midoks/mdserver-web#661] [Ideas]
主流系统和版本均已测试
上传压缩包，无法解压，会一直卡在解压界面无进展

**Comment by midoks** [comment]
什么后缀的。

**Reply by supspp** [reply]
Debian GNU/Linux 12 (bookworm) (x86_64)　 / 0180
zip,gz,等等等，所有都不行，都卡在这里，换服务器换系统也是如此，需要我提供ssh了，可以留个方式，tg登陆不进去

![QQ20241221-163941](https://github.com/user-attachments/assets/ffa57c5e-5484-4633-baee-b621632208f8)

**Reply by supspp** [reply]
已经发送邮件提交了ssh
请查看~
谢谢~

**Reply by midoks** [reply]
右键功能-解压，未移植，已修复
